caesars-palace/blocks/form/form.css

```diff
@@ -81,42 +81,46 @@ main .form.sign-in label {
   font-weight: 400;
 }
 
-main .form span.input-error {
+.form span.input-error {
   display: none;
   font-size: 11px;
   color: #d60000;
```

Comment:
Please use consistent notation for colors accross the file

Reply:
Done